<center>
    <img src="https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBMDeveloperSkillsNetwork-DS0105EN-SkillsNetwork/labs/Module2/images/SN_web_lightmode.png" width="300" alt="cognitiveclass.ai logo">
</center>


#### Add your code below following the instructions given in the course


# Data Science Tools and Ecosystem

In this notebook, Data Science Tools and Ecosystem are summarized. This is the final assigment of the course "Tools for Data Science" 10/29/2023

Some of the popular languages used in Data Scientists includes:
1. Python
2. R Languaje
3. SQL
4. Scala
5. Julia
6. JavaScript (JS)

Some of the commonly used libraries used by Data Scientists include:
If we use python the most common are:
1. Numpy
2. Matplotlib
3. Scikit-learn
4. Tensor flow

If we use R languajes the most common are:

1. ggplot
2. dplyr
3. stringr

|Data Science Tools|
|------------------|
|MySQL|
|Jupyter Notebooks|
|GitHub|
|MongoDB|

### Below are a few examples of evaluating arithmetic expressions in Python

1. Sum
2. Multiplication
3. Division

This a simple arithmetic expression to mutiply then add integers that is implemented in Python

In [1]:
(3*4)+5

17

In the following exercise is presented an arithmetic expression to convert 200 minutes to hours by diving by 60.

In [3]:
200/60

3.3333333333333335

200 minutes are 3.3333.. hours

**Objectives in this notebook**
- Use Markdowns in Jupyter Notebooks
- Remember the most common languajes and libraries used in Data Science
- Implement simple aritmethic expressions in Python
- Remeber how to make different types of markdowns in jupyter notebooks

## Made by: Marco Cevallos